### ISSUE TITLE
Extra data for training and test

### ISSUE THREAD

**Issue by anonymous-code108** [GuanRunwei/rd_map_temporal_spatial_denoising_autoencoder#1]
Hello,
The Extra data for training and test  you  provide  is .npy format, how to use it for training

**Comment by denizkumlu** [comment]
@liuning-library  have you downloaded the dataset, I could not download it? please can you help to reach dataset?

**Comment by anonymous-code108** [comment]
> @liuning-library have you downloaded the dataset, I could not download it? please can you help to reach dataset?

yes,The author provides a link to Baidu Netdisk，easy to download

**Comment by denizkumlu** [comment]
Hi Liuning, 

   I think Baidu is not working in my country properly, do you know alternative ways to download it...

thanks.

**Comment by anonymous-code108** [comment]
> Hi Liuning,
> 
> I think Baidu is not working in my country properly, do you know alternative ways to download it...
> 
> thanks.

I can send this dataset to your email

**Comment by denizkumlu** [comment]
kumlud@itu.edu.tr is my e-mail address, I will be really glad. 

Thanks.

**Comment by denizkumlu** [comment]
Hi Liuning,

    Can you share me the dataset? I shared my e-mail.

**Comment by denizkumlu** [comment]
> > Hi Liuning,
> > I think Baidu is not working in my country properly, do you know alternative ways to download it...
> > thanks.
> 
> I can send this dataset to your email

kumlud@itu.edu.tr, can you please share dataset from this e-mail?

**Comment by anonymous-code108** [comment]
> > > Hi Liuning,
> > > I think Baidu is not working in my country properly, do you know alternative ways to download it...
> > > thanks.
> > 
> > 
> > I can send this dataset to your email
> 
> [kumlud@itu.edu.tr](mailto:kumlud@itu.edu.tr), can you please share dataset from this e-mail?

I sent an email to your inbox on March 13th, attaching a link to Google Cloud

**Comment by asfriendlyascarbon** [comment]
> > > > Hi Liuning,
> > > > I think Baidu is not working in my country properly, do you know alternative ways to download it...
> > > > thanks.
> > > 
> > > 
> > > I can send this dataset to your email
> > 
> > 
> > [kumlud@itu.edu.tr](mailto:kumlud@itu.edu.tr), can you please share dataset from this e-mail?
> 
> I sent an email to your inbox on March 13th, attaching a link to Google Cloud

Could you send me the dataset too please?
It would mean a lot 

My email is asfriendlyascarbon@gmail.com
Thank you

**Comment by jaydabbb** [comment]
> 你好， 你提供的训练和测试的额外数据是.npy格式，如何使用它进行训练

你好，请问你的数据集的问题解决了吗？我现在也是用不了他的扩展数据集

**Comment by again-bin** [comment]
> > > > Hi Liuning,
> > > > I think Baidu is not working in my country properly, do you know alternative ways to download it...
> > > > thanks.
> > > 
> > > 
> > > I can send this dataset to your email
> > 
> > 
> > [kumlud@itu.edu.tr](mailto:kumlud@itu.edu.tr), can you please share dataset from this e-mail?
> 
> I sent an email to your inbox on March 13th, attaching a link to Google Cloud

Can You Send Me The Dataset Too please?
I Realy need This Dataset.

My email is lyngbn99@itsoc.sejong.ac.kr
Thank you in advance.

**Comment by anonymous-code108** [comment]
I'm sorry, but I'm not working on that direction and I don't have the dataset.Best regards



---Original---
From: ***@***.***&gt;
Date: Tue, Feb 11, 2025 13:32 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [GuanRunwei/rd_map_temporal_spatial_denoising_autoencoder] Extradata for training and test (Issue #1)




     
Hi Liuning,
 I think Baidu is not working in my country properly, do you know alternative ways to download it...
 thanks.
  
I can send this dataset to your email
  
***@***.***, can you please share dataset from this e-mail?
  
I sent an email to your inbox on March 13th, attaching a link to Google Cloud
  
Can You Send Me The Dataset Too please?
 I Realy need This Dataset.
 
My email is ***@***.***
 Thank you in advance.
